packages/ses/src/tame-date-constructor.js

```diff
@@ -69,13 +85,13 @@ export default function tameDateConstructor(dateTaming = 'safe') {
         configurable: false,
       },
       parse: {
-        value: Date.parse,
+        value: OriginalDate.parse,
```

Comment:
There is no original date. The day after New Year’s Eve, 1 BCE was New Year’s Day, 1 CE.

Reply:
What about the big bang? Or the Unix Epoch?